cost/budget_alerts/budget_alert.pt

```diff
@@ -81,33 +81,86 @@ script "js_top_level_bc", type: "javascript" do
 EOS
 end
 
+script "costs_request", type: "javascript" do
+  parameters "org","metric","bc"
+  result "request"
+  code <<-EOS
+    var date = new Date();
```

Comment:
not that you did anything wrong with formatting the date, but you might find this more efficient https://github.com/rightscale/policy_templates/blob/master/cost/scheduled_reports/scheduled_report.pt#L107-L130

Reply:
implemented